src/shared/consts.tsx

```diff
@@ -437,4 +437,6 @@ export const observerStyles = {
 
 export const enum IntegrationTypes {
   Loris = 'loris',
+  LorisUpperCase = 'LORIS', // In the API the integration type is in uppercase in some endpoints and lowercase in others
```

Comment:
I'm not seeing where we use lowercase loris, and in any case it might be better to create a separate enum for that specific usage. I think all the new endpoints are using `LORIS`.

Reply:
My question was more related to which one is using lowercase? I know the backend expects uppercase.